### ISSUE TITLE
Different view of the header of the table during editing and after saving

### ISSUE THREAD

**Issue by petrhusak** [Zavy86/WikiDocs#186]
### Describe your issue

In the preview while page editing, the header is part of the table. But when I save the changes, the header appears above the table, making it look like it's not part of the table.

### Device and settings

Linux Mint, Chrome, Firefox

### Steps to reproduce

Self-explaining, try to create page with tahble and click on "Toggle preview" button.

### Screenshots (optional)

This is how table looks like in preview while editing:
![tbl_editing](https://github.com/user-attachments/assets/fdc7382e-d0c7-47a4-85b2-341478891ebf)
and after saving:
![tbl_saved](https://github.com/user-attachments/assets/19a8ec1e-1afe-428e-a62e-02e05006e47d)

### Extra fields

- [ ] I'd like to work on this issue

**Comment by petrhusak** [comment]
I noticed that the header is also displayed differently if it is empty.
While editing:
![tbl_edit](https://github.com/user-attachments/assets/9772baf1-cdff-46cd-a942-747b9e83815a)
and after saving:
![tbl_saved](https://github.com/user-attachments/assets/17388855-ab48-487e-9e7f-8327d61b0a9b)
It looks like there is standard markdown in the preview and after saving wikidocs uses "tweaked" markdown (bcs hidden header like this is cool, better than standard markdown :+1: ).
So, maybe the case I described as a bug is just a feature of how the tables are displayed in wikidocs :slightly_smiling_face: (even though in case with filled header I prefer standard Markdown).

However, I would like to see in the preview exactly how it will be displayed after saving.
